canonicalwebteam/templates/image_template.html

```diff
@@ -4,16 +4,21 @@
               {% if width > 720 %},https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto,w_720/https://assets.ubuntu.com/{{ path }} 767w{% endif %}
               {% if width > 990 %},https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto,w_990/https://assets.ubuntu.com/{{ path }} 1030w{% endif %}"
   data-src="https://res.cloudinary.com/canonical/image/fetch/q_auto,f_auto/https://assets.ubuntu.com/{{ path }}"
-  sizes="100vw"
-  alt="test"
+  data-sizes="100vw"
```

Comment:
Why is `sizes` needed? It defaults to `100vw` anyway. I think we're better off leaving it as default if we don't have a reason to override it.